include/loop_tool/mutate.h

```diff
@@ -50,8 +55,13 @@ LoopTree maximize_reuse(const LoopTree& lt);
 LoopTree unroll_inner_loops(const LoopTree& lt, int32_t unroll_amount);
 
 // Informational functions
-int64_t flops(const LoopTree& lt);
+double eval_runtime(const LoopTree &lt);
```

Comment:
now that I think about it, it's not entirely clear from the name what this measures.  maybe "benchmark_flops" or something?

Reply:
I changed flops to FLOPs, and added FLOPS as FLOPs / seconds. eval_runtime is not the best name eighter, maybe something like time_seconds would make more sense.